app/models/submission.rb

```diff
@@ -196,4 +196,10 @@ def name
   def phone
     user_phone || user&.phone
   end
+
+  def count_submissions_of_user
+    return Submission.where(user_email: user_email).count if user.nil?
```

Comment:
What do you think about finding submissions exclusively by the same emails?

Reply:
I suppose It should also ignore non anonymous submissions.

Reply:
If our submission is anonymous, then a user is not created for it

Reply:
Resolved

Reply:
I mean that if the submission is anonymous you should search among other anonymous submissions instead of all of their

Reply:
I understand, but now it doesn't matter anymore :)